app/models/question_set.rb

```diff
@@ -78,6 +78,10 @@ def question(question_identifier, required: true)
     result
   end
 
+  def summary_questions
```

Comment:
@meltheadorable I don't like this particular method. If you have a suggested improvement here, please let me know. The code only needs to check if there are questions with the display_in_summary attribute set to `true`. The code is not using that list to questions to populate any content.

Reply:
There may be another option here involving scopes, but if this is really only checking whether they exist, you should be able to do

```ruby
def summary_questions?
  questions.where(display_in_summary: true).count > 0
end
```

which should do a much smaller/faster query, and would avoid you needing to do the length check later in the view.

Reply:
Thank you!